assets/pages/monthly-contributions-thankyou/monthlyContributionsThankyou.jsx

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+// ----- Imports ----- //
+
+import 'ophan';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import SimpleHeader from 'components/headers/simpleHeader/simpleHeader';
+import SimpleFooter from 'components/footers/simpleFooter/simpleFooter';
+
+import * as ga from 'helpers/ga';
+import * as abTest from 'helpers/abtest';
+import * as logger from 'helpers/logger';
+
+
+// ----- AB Tests ----- //
```

Comment:
We should extract this common chunk asap in a helper module. We now have this piece of code 3 times across the site.